docs/pages/get-started/create-a-project.mdx

```diff
@@ -17,7 +17,7 @@ To create a new project, run the following command:
 
 <Terminal cmd={['$ npx create-expo-app@latest']} />
 
-> The default project template includes [Expo Router](/router/introduction/) library installed to build multi-screen apps with Expo. You can choose a different template by using the `--template` option. For example, using this option, you can choose the `blank` template to create a project with minimum required libraries installed and without navigation configured.
+> You can choose a different template by adding the [`--template` option](/more/create-expo/#--template).
```

Comment:
cc @jonsamp reverting this change back to the original one since we now have `--template` option documented and a user can look at the link for detailed info.